### DISCUSSION TITLE
Is a way to run focused puppeteer tests?

### DISCUSSION THREAD

**Discussion by erichermann** [storybookjs/storybook#14277] [Help]
Hello, 
    We're using puppeteer with Storybook and our puppeteer test suite is getting quite large and cumbersome to write new test.   I was wondering if there was a way to run individual puppeteer test files or, even better run a single puppeteer test.

Here's our Storybook/Puppeteer configuration which lives in src/__tests__/storyshots-puppeteer.test.js:
```
initStoryshots({
  suite: 'Puppeteer storyshots',
  test: puppeteerTest()
});
```

This is how we're running our tests:
```
npm test -- src/__tests__/storyshots-puppeteer.test.js
````

It would be nice to to be able to do something like:

```
npm test -- src/__tests__/storyshots-puppeteer.test.js --file src/stories/product-form.stories.js
````

**Comment by WallisonLima** [comment]
Não seria mais viável dividir as funções de testes do tireteiro em funções externas, executando em mais de um terminal?
Apenas copie as funções que seriam compartilhadas nos mesmos tireteiros.
Para te ajudar preciso de mais detalhes.
